netlify.toml

```diff
@@ -0,0 +1,11 @@
+# Global settings applied to the whole site.  
+# 
+# “publish” is the directory to publish (relative to root of your repo),
+# “command” is your build command,
+# “base” is directory to change to before starting build. if you set base:
+#    that is where we will look for package.json/.nvmrc/etc not repo root!
+
+[build]
+  base    = "./starwars"
+  publish = "./starwars/build"
+  command = "yarn build"
```

Comment:
Nice work hosting your site on netlify! Thanks for sharing these helpful directions with everyone 😊